contrib/ansible/openshift/README.md

```diff
@@ -0,0 +1,18 @@
+## Pbench Ansible
+Playbook to register pbench tools on openshift cluster
+
+### Requirements
+You need to have these installed
+   - Openshift
+   - pbench
+
+### Run
+```
+$ ansible-playbook -i /path/to/inventory pbench_register.yml
```

Comment:
could you please update this to point out the fact that it is designed to work with the openshift-ansible inventory file?